.github/ci/generate-news-feed.js

```diff
@@ -0,0 +1,113 @@
+import { Feed } from 'feed';
+import fs from 'fs';
+
+/**
+ * @type {FeedConfig[]}
+ */
+const feeds = [
+  {
+    title: '24life Updates',
+    targetFile: `../../feed.xml`,
+    source: 'https://main--24life--hlxsites.hlx.live/feed.json',  
```

Comment:
@synox, I'm not sure what the source should be?  should it be https://www.24life.com/feed/?